### PR TITLE
W-13222293/adopt amf scala js1

### PR DESCRIPTION
**Pull request by mansillafederico**
[W-132222931](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001RdGAAYA3/view): Adopt AMF versions, pull develop-scalaJS

### CODE REVIEW

**Review by BlockLucas — CHANGES_REQUESTED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -148,13 +148,14 @@ pipeline {
                     branch 'master'
                     branch 'develop'
                     branch 'rc/*'
+                    branch 'develop-SCALAJS1'
```
[the remark this answers is not in the document]
Comment (mansillafederico) [reply Jenkinsfile:151]: Done

File: Jenkinsfile [hunk 2]
```diff
@@ -198,14 +200,15 @@ pipeline {
...
```
[the remark this answers is not in the document]
Comment (mansillafederico) [reply Jenkinsfile:203]: Done

File: als-actions/js/package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aml-org/amf-antlr-parsers": "0.6.23",
+    "@aml-org/amf-antlr-parsers": "0.6.22-SCALAJS1.6-SNAPSHOT",
```
[the remark this answers is not in the document]
Comment (mansillafederico) [reply als-actions/js/package.json:9]: This is replaced when build with sbt but I can replace it anywhere

[the remark this answers is not in the document]
Comment (mansillafederico) [reply als-actions/js/package.json:9]: Done

File: als-common/js/package.json [hunk 4]
```diff
@@ -6,7 +6,7 @@
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "dependencies": {
-    "@aml-org/amf-antlr-parsers": "0.6.23",
+    "@aml-org/amf-antlr-parsers": "0.6.22-SCALAJS1.6-SNAPSHOT",
```
[the remark this answers is not in the document]
Comment (mansillafederico) [reply als-common/js/package.json:9]: Done

File: als-common/shared/src/main/scala/org/mulesoft/amfintegration/AbstractDeclarationInformation.scala [hunk 5]
```diff
@@ -53,7 +54,7 @@ object AbstractDeclarationInformation {
                 keepEditingInfo = false,
                 UnhandledErrorHandler,
                 amfConfiguration,
-                BaseUnitSourceLocationIndex.build(bu)
+                BaseUnitSourceLocationIndex.build(bu) // This line should be uncommented when AMF sacalaJS is released
               )
             extendsHelper.parseOperation(bu, t.name.option().getOrElse(""), "AbstractDeclarationInformation", e)
           })
```
[the remark this answers is not in the document]
Comment (mansillafederico) [reply als-common/shared/src/main/scala/org/mulesoft/amfintegration/AbstractDeclarationInformation.scala]: Comment removed

**Review by BlockLucas — APPROVED** [review]
LGTM

**Pull request merged**